spec/mailers/grant_creator_request_mailer_spec.rb

```diff
@@ -7,25 +7,44 @@
     let(:request) { create(:grant_creator_request) }
     let(:mailer)           { described_class.system_admin_notification(request: request)}
 
-    it 'has the appropriate subject' do
-      expect(mailer.subject).to eql "Pending Grant Creator Requests in #{COMPETITIONS_CONFIG[:application_name]}"
-    end
 
-    it 'uses the system admins\' email addresses' do
-      email = system_admin.email
-      expect(mailer.to).to include email
-    end
+    context 'system admin does not exists'  do
+      it 'has an empty mailer' do
+        expect(mailer.subject).to be_falsey
+        expect(mailer.bcc).to be_falsey
+        expect(mailer.body).to be_empty
+      end
 
-    it 'includes a grant requester\'s name' do
-      expect(mailer.body).to have_link full_name(request.requester), href: grant_creator_request_url(request)
+      it 'does not throw an exception' do
+        expect {mailer}.not_to raise_error
+      end
     end
 
-    it 'includes the application root url' do
-      expect(mailer.body).to include COMPETITIONS_CONFIG[:application_name]
-    end
+    context 'system admin exists' do
+      before(:each) do
+        system_admin
+      end
+
+      it 'has the appropriate subject' do
+        expect(mailer.subject).to eql "Pending Grant Creator Requests in #{COMPETITIONS_CONFIG[:application_name]}"
+      end
+
+      it 'uses the system admins\' email addresses' do
+        email = system_admin.email
+        expect(mailer.to).to include email
+      end
+
+      it 'includes a grant requester\'s name' do
+        expect(mailer.body).to have_link full_name(request.requester), href: grant_creator_request_url(request)
+      end
+
+      it 'includes the application root url' do
+        expect(mailer.body).to include COMPETITIONS_CONFIG[:application_name]
+      end
 
-    it 'includes a link to the grant creator requests index page' do
-      expect(mailer.body).to have_link grant_creator_requests_url.to_s, href: grant_creator_requests_url
+      it 'includes a link to the grant creator requests index page' do
+        expect(mailer.body).to have_link grant_creator_requests_url.to_s, href: grant_creator_requests_url
+      end
     end
```

Comment:
Add spec for when there are no recipients?

Reply:
It should have one. 
https://github.com/NUARIG/competitions/blob/5323466ae2a69b16896b03cf7310d7faef9e79bf/spec/mailers/grant_creator_request_mailer_spec.rb#L11